changelog/58713.md

```diff
@@ -0,0 +1,3 @@
+**fix:**
```

Comment:
the changelog filename needs to follow this format: https://docs.saltstack.com/en/master/topics/development/changelog.html#how-do-i-add-a-changelog-entry

and doesn't need `**fix**` inside the text file. Also since this change does not align with one of the types listed there you do not need a changelog entry since you are not fixing a bug and simply adding a test, so you can just remove this file for now.

Reply:
File removed

Reply:
Hmm .. looks like the pre-commit now forces a changelog entry for all commits?
```
Check Changelog Entries..................................................Failed
- hook id: check-changelog-entries
- exit code: 1
```

Reply:
I'm not sure why that was not allowing you to remove the file, but I went ahead and removed it for you. We did recently add a pre-commit check to ensure changelog entries are in the correct path if they exist, but does not require you to have a changelog entry.